test/tests/p5.Envelope.js

```diff
@@ -0,0 +1,56 @@
+let expect = chai.expect;
+
+describe('p5.Envelop', function () {
+  let sf;
+
+  it('can be created and disposed', function () {
+    let envelope = new p5.Envelope();
```

Comment:
```suggestion
    const envelope = new p5.Envelope();
```

We can use `const` instead of `let` for variables that won't be re-assigned within their scope

Reply:
Done